src/components/EditProfileForm.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import TextField from './common/TextField';
+import TextArea from './common/TextArea';
+import ErrorAlertNotification from './common/ErrorAlertNotification';
+
+import EditProfileValidate from '../middlewares/EditProfileValidate';
+
+/**
+ * @EditProfileForm page
+ */
+export class EditProfileForm extends Component {
+  /**
+   * constructor
+   *
+   * @return {void}
+   */
+  constructor() {
+    super();
+    this.state = {
+      image: '',
+      firstname: '',
+      lastname: '',
+      email: '',
+      bio: '',
+      errors: {},
+      error: {},
+    };
+
+    this.onChange = this.onChange.bind(this);
```

Comment:
Please use arrow functions and try to avoid passing state through constructors. This is to prevent using bind(this).